src/components/select/filterable-select/filterable-select.spec.js

```diff
@@ -112,22 +112,22 @@ describe("FilterableSelect", () => {
 
     it("the SelectList should not be rendered", () => {
       wrapper.find("input").simulate("focus");
-      expect(wrapper.find(SelectList).exists()).toBe(false);
+      expect(wrapper.find(SelectList).prop("isOpen")).toBe(false);
```

Comment:
For these tests let's assert the styling is or isn't applied (using `toHaveStyleRule`) here rather than the prop value